_docs/_latest/use/explain.md

```diff
@@ -0,0 +1,270 @@
+
+To constrain the result to a certain type of permission, use the following command:
+
+```
+$ forseti-client-XXXX-vm> forseti explainer access_by_member user/<USER_NAME> iam.roles.get
+```
+
+The above query will list all resources that can be accessed by the member with a binding of `iam.roles.get`.
+
+#### List resource permissions
+
+The following command lists all members that can access a resource by a direct binding:
+
+```
+$ forseti-client-XXXX-vm> forseti explainer access_by_resource <RESOURCE_NAME>
+```
+
+Similar to the `access_by_member` command, group expansion is not performed by default. For example, `group/bar` has `roles/owner` on the `organization/1234567890` and no other policies. Without group expansion, the following example will only return `group/bar`:
+
+```
+$ forseti-client-XXXX-vm> forseti explainer access_by_resource organizations/1234567890
+```
+
+To enable group expansion, pass in the `--expand_group` argument. Forseti will perform group expansion and list all members that can access the resource, even if the access is granted because a member is in a group.
+
+```
+$ forseti-client-XXXX-vm> forseti explainer access_by_resource <RESOURCE_NAME> --expand_group true 
+```
+
+With group expansion enabled, the example above will list `group/bar` and `user/foo` if `user/foo` is a member in `group/bar`.
+
+To constrain the result to a certain type of permission, pass in the permission:
+
+```
+$ forseti-client-XXXX-vm> forseti explainer access_by_resource <RESOURCE_NAME> <PERMISSION> 
+```
+
+#### Access member or resource by permission
+
+You can specify a permission, such as `iam.serviceAccounts.get`, and list all the `<members, resource, role>` that have a relation.
+
+The tuple `<members, resource, role>` contains the members with a role that contains permission `iam.serviceAccounts.get` on the resource.
+
+```
+$ forseti-client-XXXX-vm> forseti explainer access_by_authz --permission iam.serviceAccounts.get
+...
+Member member1
+Member member2
+Resource resource1
+Role roles/owner
+...
+```
+
+These results mean that resource `resource1` has a policy to grant `roles/owner` to `member1` and `member2`
+
+With `access_by_authz` you can also specify a role instead of a permission:
+
+```
+# With G Suite Group expansion
+$ forseti-client-XXXX-vm> forseti explainer access_by_authz --permission iam.serviceAccounts.get --expand_group
+
+# With resource expansion.
+$ forseti-client-XXXX-vm> forseti explainer access_by_authz --permission iam.serviceAccounts.get --expand_resource
+```
+
+#### View permission source
+Understand why a member has a permission to a resource:
+
+```
+$ forseti-client-XXXX-vm> forseti explainer why_granted <MEMBER_NAME> <RESOURCE_NAME> --permission <PERMISSION_NAME>
+```
+
+Example values for `<MEMBER_NAME>`, `user/user1@gmail.com`, `serviceAccount/service1@domain.com`
```

Comment:
Can you explain how to interpret the output?

Reply:
Added example output